deps/checksums/llvm-6.0.0.src.tar.xz/md5

```diff
@@ -0,0 +1 @@
+788a11a35fa62eb008019b37187d09d2
```

Comment:
Maybe rebase onto master which already should have these files (You should not check them int)